pkg/util/hardware/hardware_info.go

```diff
@@ -105,6 +106,11 @@ func GetFreeMemoryCount() uint64 {
 
 // GetDiskUsage Get Disk Usage in GB
 func GetDiskUsage(path string) (float64, float64, error) {
+	// Check if the path exists, skip get disk usage if not exists
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
why don't we catch os.IsNotExit error  on usage.
Os.Stat add another filesystem call which is not nessary.

Reply:
The local storage path does not exist if mmap is disabled, meaning there will be no available data path to check.